src/EnergyPlus/SystemAvailabilityManager.cc

```diff
@@ -2301,29 +2290,64 @@ namespace SystemAvailabilityManager {
 		int I;
 		int J;
 		Real64 TempDiff;
-		static Real64 TempDiffHi( 0.0 );
-		static Real64 TempDiffLo( 0.0 );
-		//  LOGICAL, ALLOCATABLE, SAVE, DIMENSION(:) :: ZoneCompOptStartControlType
+		Real64 TempDiffHi;
+		Real64 TempDiffLo;
```

Comment:
Must keep track of the result by air loop num since this calc is used the next time through.